containers/datalab/Dockerfile.in

```diff
@@ -107,11 +107,16 @@ RUN mkdir -p /datalab/web && \
     cd / && \
     /tools/node/bin/npm install -g forever
 
-# Install ProtoBuf, TensorFlow, DataFlow
+# Install ProtoBuf, TensorFlow
 RUN pip install protobuf==3.0.0b2.post2 && \
-    wget https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0-cp27-none-linux_x86_64.whl && \
-    pip install tensorflow-0.8.0-cp27-none-linux_x86_64.whl && \
-    rm tensorflow-0.8.0-cp27-none-linux_x86_64.whl
+    wget https://storage.googleapis.com/cloud-datalab/deploy/tf/tensorflow-0.9.0-cp27-none-linux_x86_64.whl && \
+    pip install tensorflow-0.9.0-cp27-none-linux_x86_64.whl && \
+    rm tensorflow-0.9.0-cp27-none-linux_x86_64.whl
+
+# Install DataFlow
+RUN wget https://github.com/GoogleCloudPlatform/DataflowPythonSDK/archive/v0.2.7.tar.gz && \
+    pip install v0.2.7.tar.gz && \
+    rm v0.2.7.tar.gz
```

Comment:
We should keep the source code (either as a tarball or as the extracted directory) in the built image.